bot/exts/info/source.py

```diff
@@ -36,7 +36,8 @@ async def convert(self, ctx: commands.Context, argument: str) -> SourceType:
             return argument.lower()
 
         raise commands.BadArgument(
-            f"Unable to convert `{argument}` to valid command{', tag,' if show_tag else ''} or Cog."
+            f"Unable to convert `{utils.escape_markdown(argument)}` to valid\
+                 command{', tag,' if show_tag else ''} or Cog."
         )
```

Comment:
Prevailing style to create a multi-line string would be:
```python
raise commands.BadArgument(
    "This is line 1 "
    "followed by line 2."
)
```
But you could also build the escaped `argument` separately and store it in a shorter-named variable.